lib/services/project-changes-service.ts

```diff
@@ -227,11 +230,9 @@ export class ProjectChangesService implements IProjectChangesService {
 
 			let fileStats = this.$fs.getFsStats(filePath);
 
-			let changed = fileStats.mtime.getTime() >= this._outputProjectMtime || fileStats.ctime.getTime() >= this._outputProjectCTime;
-			if (!changed) {
-				let lFileStats = this.$fs.getLsStats(filePath);
-				changed = lFileStats.mtime.getTime() >= this._outputProjectMtime || lFileStats.ctime.getTime() >= this._outputProjectCTime;
-			}
+			let changed =  fileStats.mtime.getTime() >= this._outputProjectMtime ||
```

Comment:
You got a point, but we're safe because the API is synchronous. I'll change it anyway, though, just in case.